package.json

```diff
@@ -56,7 +56,7 @@
     "joi": "^17.4.0",
     "js-yaml": "^3.14.1",
     "keymbinatorial": "^1.1.6",
-    "screwdriver-data-schema": "^21.2.4",
+    "screwdriver-data-schema": "../data-schema",
```

Comment:
TODO: Update the version after https://github.com/screwdriver-cd/data-schema/pull/460 is merged